robot/markers.py

```diff
@@ -83,20 +68,15 @@ class Marker:
     def __init__(self, data):
         self._raw_data = data
 
-        # Go through all the data, add an _ at the start.
-        data = {"_" + k: v for k, v in data.items()}
-
-        self.__dict__.update(data)
-
     @property
-    def id(self):
+    def id(self) -> int:
         """ID of the marker seen."""
-        return self._id
+        return self._raw_data['id']
 
     @property
-    def size(self):
+    def size(self) -> Tuple[TMetres, TMetres]:
         """Marker size in metres."""
-        return tuple(self._size)
+        return self._raw_data['size']
```

Comment:
Does this not need conversion to a tuple? (I'm assuming that it's a list by default given that it's decoded from JSON)

Reply:
Hnng, it's worse than that -- there is no `'size'` key in this dictionary. Ughfg. I've opened #60 to remove this which we can merge later.